src/Components/SysTable/SysTable.js

```diff
@@ -10,17 +26,11 @@ import { gqlProps } from '../Common';
 import messages from '../../Messages';
 import { RegistryContext } from '../../store';
 import { globalFilters } from '../../store/cache';
+import { useDispatch } from 'react-redux';
 import matchFilter from './matchFilter';
-import { buildActiveFiltersConfig, buildApiFilters } from './Helpers';
-import { useFetchSystems } from './hooks';
-import {
-  defaultOnLoad,
-  columns,
-  mergedColumns,
-  appendDirection,
-  findColumnByKey,
-} from './constants';
-import './SysTable.scss';
+import { buildActiveFiltersConfig } from './Helpers';
```

Comment:
This is probably the reason you are getting the LOAD_ENTITIES error. `buildApiFilters` is in Helpers.js not on `hooks`